runtime/mangata-rococo/src/lib.rs

```diff
@@ -503,7 +505,7 @@ impl pallet_xyk::Config for Runtime {
 	type PoolFeePercentage = frame_support::traits::ConstU128<20>;
 	type TreasuryFeePercentage = frame_support::traits::ConstU128<5>;
 	type BuyAndBurnFeePercentage = frame_support::traits::ConstU128<5>;
-	type RewardsDistributionPeriod = frame_support::traits::ConstU32<1200>;
+	type RewardsDistributionPeriod = SessionLenghtOf<Runtime>;
```

Comment:
is this correct ? shouldn't this be the number of blocks ? 
new value of 4 * HOURS != old value of 1200
SessionLenghtOf<Runtime> / 12 ?
maybe modify the pallet_xyk to use SessionLength and compute the num of blocks internally ?

Reply:
https://github.com/mangata-finance/mangata-node/blob/96f92d66b0bdb8f03684c2a1f80f97ca8b6dfbf4/runtime/mangata-rococo/src/lib.rs#L213-L215

HOURS is already defined in blocks 

1 HOUR = 300 blocks
4 HOURS = 1200 blocks

Reply:
aaaaaaaa :) sorry